src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,40 @@
+package calculator
+
+import calculator.Operator.Companion.toOperator
+
+class Calculator {
+    fun calculate(input: String?): Double {
+        if (input.isNullOrBlank()) throw IllegalArgumentException("계산할 수 없는 입력값입니다.")
```

Comment:
kotlin에서 제공하는 [require 함수](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html)를 활용해볼 수 있을 것 같아요.  😃
```suggestion
        require(!input.isNullOrBlank()) { "계산할 수 없는 입력값입니다." }
```

Reply:
아하, 관련 문서 읽고 적용해보겠습니다!